packages/primitives/progress-bar/license.md

```diff
@@ -0,0 +1,21 @@
+# The MIT License (MIT)
```

Comment:
I like this approach more than expanding the Block primitive 👍 It's very clear and I see you managed to cut out like 90% of the code that was used in Block, so it doesn't feel like duplication at all.